json-rpc/API-CHANGELOG.md

```diff
@@ -13,6 +13,11 @@ Please add the API change in the following format:
 
 ```
 
+## 2020-12-09 Add `get_events_with_proof` APIs
```

Comment:
could you update this date to the date you land before you land this PR, as we have release 1.1 cut recently, it's better to use the date you are going to land.